components/GetRemoteRuntimes.coffee

```diff
@@ -0,0 +1,26 @@
+noflo = require 'noflo'
+registry = require 'flowhub-registry'
+
+exports.getComponent = ->
+  c = new noflo.Component
+  c.inPorts.add 'user',
+    datatype: 'object'
+  c.outPorts.add 'runtime',
+    datatype: 'object'
+  c.outPorts.add 'error',
+    datatype: 'object'
+
+  noflo.helpers.WirePattern c,
+    in: 'user'
+    out: 'runtime'
+    async: true
+  , (data, groups, out, callback) ->
+    return unless data['grid-token']
+    registry.list data['grid-token'],
+      host: '$NOFLO_REGISTRY_SERVICE'
+    , (err, runtimes) ->
+      return callback err if err
+      out.beginGroup '$NOFLO_REGISTRY_SERVICE'
```

Comment:
Can we please get the right string from a config file/module? Instead of spreading our reliance on string-replacement of source code